models/ed/R/write.configs.ed.R

```diff
@@ -363,14 +363,13 @@ write.config.ED2 <- function(trait.values, settings, run.id, defaults = settings
   if (!is.null(custom_tags)) {
     # Convert numeric tags to numeric
     # Anything that isn't converted to NA via `as.numeric` is numeric
-    try_numeric <- suppressWarnings(vapply(custom_tags, as.numeric, numeric(1)))
-    are_numeric <- !is.na(try_numeric)
-    custom_tags[are_numeric] <- lapply(custom_tags[are_numeric], as.numeric)
+    custom_tags <- lapply(custom_tags,
+                          tryCatch(as.numeric(x), warning = function(e) x))
```

Comment:
The `tryCatch(...)` is a more elegant version of what I had on the left. Using `lapply` instead of `vapply` makes this robust to `custom_tags` that are vectors.